Make trivial rule for particle.io

jish opened the pull request:
Particle uses quite a few subdomains and most appear to support HTTPS.
`community` in particular, needs a redirect rule as it does not
redirect to HTTPS automatically.

J0WI commented:
Thanks for adding this!

Could you add comments on the top, that blog.particle.io and status.particle.io are serving a mismatching cert?
projects.particle.io also works with https.

jish commented:
I looked through some of the other files to get a sense of the comment style. I added this comment to the top of the file. If that is not correct, or if you have a guide I can look at, then I can adjust the comments.

```
<!--

	Problematic domains:

		- blog.particle.io	(mismatched)
		- status.particle.io	(mismatched)

-->
```

I also added a `<target>` tag for `projects.particle.io`.

J0WI commented:
Thanks!
